### ISSUE TITLE
vararg pipeops

### ISSUE THREAD

**Issue by mb706** [mlr-org/mlr3pipelines#85]
use `...` as name in the intype list to identify a function with arbitrary number of inputs.

**Comment by mb706** [comment]
Problem currently:
```r
g = PipeOpBranch$new(2) %>>%
  gunion(list(mlr_pipeops$get("scale"), mlr_pipeops$get("pca"))) %>>%
  PipeOpUnbranch$new(2)
```
Suggested usage:
```r
g = PipeOpBranch$new(2) %>>%
  gunion(list(mlr_pipeops$get("scale"), mlr_pipeops$get("pca"))) %>>%
  mlr_pipeops$get("unbranch")
```

**Comment by berndbischl** [comment]
why cant we get rid of both 2s at the same time ?

**Comment by mb706** [comment]
A `PipeOp` function can easily find out how many inputs it is getting: `length(inputs)`. Outputs is a bit more annoying; we can't set it in the `PipeOp` because it could theoretically be member of multiple `Graph`s. We can give it the number of supposed outputs as an argument in the function call, or give it a reference to the `Graph` in the function call, but this breaks the very nice encapsulation we have right now and makes things quite a bit uglier imho.

**Comment by mb706** [comment]
Closed by #186
